src/emulator/functionsEmulator.ts

```diff
@@ -828,7 +821,9 @@ export function InvokeRuntime(
 
   return {
     exit: new Promise<number>((resolve) => {
-      runtime.on("exit", resolve);
```

Comment:
This change seems like a no-op?

Reply:
Yeah you're right I was in there debugging